modules/vector-search/pages/create-vector-search-index-ui.adoc

```diff
@@ -77,9 +80,13 @@ For more information, see xref:search:child-field-options-reference.adoc[].
 For more information, see xref:search:child-field-options-reference.adoc[].
 .. Select *Index*.
 .. Click btn:[OK].
-. (Optional) xref:search:create-child-field.adoc[Create another child field] on the new collection type mapping for any additional fields you want to return in your search results.
+
++
+For more information about how to create child fields, see xref:search:create-child-field.adoc[].
+. (Optional) Create another child field on the new collection type mapping for any additional fields you want to return in your search results.
 +
 For example, you could add the text field that you used to generate your vector embeddings. 
+For more information about how to create child fields, see xref:search:create-child-field.adoc[].
```

Comment:
```suggestion

```

Reply:
I believe this is a repeated sentence? You have this sentence on line 85.

Reply:
Nope, I want this here, too, since they need to create another child field in this step, too.

Reply:
Ok noted.